docs/Metrics-System.md

```diff
@@ -56,15 +56,32 @@ The following shows the details of the available metrics.
 * UnderFsCapacityFree: Free capacity of the under file system in bytes.
 * Workers: Number of the workers.
 * FilesTotal: Total number of files and directories in the file system.
-* FilesCheckpointed: Total number of the files checkpointed.
-* FilesCreated: Total number of files and directories created.
-* FilesDeleted: Total number of files and directories deleted.
+* FilesPersisted: Total number of the files persisted.
```

Comment:
Let's divide the metrics into logical groups (e.g. # of API operations vs. internal counts) and within each logical group, sort the entries alphabetically.